services/cards/src/components/TopBar.tsx

```diff
@@ -61,26 +62,30 @@ class TopBar extends React.Component<Props, {}> {
       );
     });
 
-    return (
-      <AppBar>
-        <Toolbar className="printHide">
-          <Typography variant="title">
-            Expedition
-          </Typography>
-          {filters}
-          <Tooltip title="Reload card data">
-            <IconButton onClick={this.props.downloadCards}>
-              <AutoRenew />
-            </IconButton>
-          </Tooltip>
-          <Tooltip title="Help">
-            <IconButton onClick={this.props.openHelp}>
-              <HelpOutline />
-            </IconButton>
-          </Tooltip>
-        </Toolbar>
-      </AppBar>
-    );
+    if (this.props.printing) {
+      return null;
+    } else {
```

Comment:
This doesn't need to be in an `else` bracket if the initial conditional returns early.